pkg/controller/podautoscaler/horizontal.go

```diff
@@ -953,9 +953,9 @@ func (a *HorizontalController) normalizeDesiredReplicas(hpa *autoscalingv2.Horiz
 	desiredReplicas, condition, reason := convertDesiredReplicasWithRules(currentReplicas, stabilizedRecommendation, minReplicas, hpa.Spec.MaxReplicas)
 
 	if desiredReplicas == stabilizedRecommendation {
-		setCondition(hpa, autoscalingv2.ScalingLimited, v1.ConditionFalse, condition, reason)
+		setCondition(hpa, autoscalingv2.ScalingLimited, v1.ConditionFalse, condition, "%s", reason)
```

Comment:
nit:
there are so many "%s" format being added to `setCondition` and `framework.Logf`.  it looks kind of redundant. do we have any other way to make `go vet` happy? maybe create a new `setConditionF`?

Reply:
I would avoid creating new helper functions in this cleanup... if the only helper we have takes a format string and args, `"%s", ...` is fine for now

For helpers which have a non-format alternative (like `fmt.Errorf() → errors.New()`), we should use that